pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx

```diff
@@ -60,11 +61,13 @@ import './spaceConfiguration.scss';
 
 const IsValidName = ZodMatcher(SpaceBaseInfoSchema.shape.name);
 const IsValidDescription = ZodMatcher(SpaceBaseInfoSchema.shape.description);
+const IsValidFluffText = ZodMatcher(SpaceBaseInfoSchema.shape.fluffText);
 
 function DefaultConfig(): SpaceDirectoryConfig {
 	return {
 		name: '',
 		description: '',
+		fluffText: '',
```

Comment:
why is it called `fluffText` I wouldn't be able to guess what it is used for unless I also find the label for the user input, maybe it could be renamed to something more intuitive like:

`welcomeText`

Reply:
Oh that's just me. "Fluff text" is text used on board games to fluff out game components without addin new mechanics or rules. But I understand that it is confusing and will change it.